angular-client/src/services/socket.service.ts

```diff
@@ -24,6 +24,7 @@ export default class SocketService {
     this.socket.on('message', (message: string) => {
       try {
         const data = JSON.parse(message) as ServerData;
+        storage.setCurrentRunId(data.runId);
```

Comment:
so storage is just acting as a cache?

Reply:
Its also where we access all the most recent real time data points